src/SAML2/Protocol/Saml20MetadataHandler.cs

```diff
@@ -57,9 +57,10 @@ public override void ProcessRequest(HttpContext context)
                     throw new ArgumentException(ErrorMessages.MetadataSignQueryParameterInvalid);
                 }
             }
-                        
-            context.Response.ContentType = Saml20Constants.MetadataMimetype;
-            context.Response.AddHeader("Content-Disposition", "attachment; filename=\"metadata.xml\"");
+
+            //context.Response.ContentType = Saml20Constants.MetadataMimetype;
+            context.Response.ContentType = "text/xml";
```

Comment:
What was the problem with Saml20Constants.MetadataMimetype (application/samlmetadata+xml)?